### PR TITLE
Feature/show speaker icon

### LINKED ISSUE #115: Show speaker icon for droidkaigi fm
## Kind (Required)

- Improvement

 ## Overview (Required)

- We can use icons!  image
 - design
image

 ## Links	

- https://www.figma.com/file/IFlrbfmBSdYvUz7VmSzfLV/DroidKaigi_2021_official_app?node-id=0%3A1

### PR DESCRIPTION
**Pull request by KoutaMatsushita**
## Issue
- close #115

### design
- show speakers icon.
- resize title. (with reference to proposed design.)
### data
- get speakers by api.
- fix `GenerateFakeContents`.
  - for `List<Speaker>` serialize support.
- update `fakeFeedApi`.
  - `http https://ssot-api-staging.an.r.appspot.com/feeds/recent | jq | pbcopy`

## Links
- [design](https://www.figma.com/file/IFlrbfmBSdYvUz7VmSzfLV/DroidKaigi_2021_official_app?node-id=0%3A1)

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
👀

File: data/api/src/androidTest/kotlin/io/github/droidkaigi/feeder/data/GenerateFakeContents.kt [hunk 1]
```diff
@@ -17,7 +17,7 @@ class GenerateFakeContents {
             "fun fakeFeedContents(): FeedContents {\n" +
             "    return FeedContents(\n" +
             "        // Generated by GenerateFakeContents.kt\n" +
-            "        feedContents = listOf(" + build(feedList) + "),\n" +
+            "        feedItemContents = " + build(feedList) + ",\n" +
```
Comment (takahirom) [inline data/api/src/androidTest/kotlin/io/github/droidkaigi/feeder/data/GenerateFakeContents.kt:20]: Thank you for updating this process 🙏

File: model/src/commonMain/kotlin/io/github/droidkaigi/feeder/FeedContents.kt [hunk 2]
```diff
@@ -50,17 +48,13 @@ fun fakeFeedContents(): FeedContents {
...
                 summary = MultiLangText(
-                    jaTitle = "DroidKaigiスタッフの@tomoya0x00です。DroidKaigi " +
-                        "2020の開催日が近づいてきましたね。この記事では、DroidKaigi 2020 Day.1に実施する " +
-                        "DroidKaigiオリジナルCodelabsについてご紹介します。",
-                    enTitle = "This is tomoya0x00 from DroidKaigi Committee. DroidKaigi 2020 is " +
-                        "approaching!This post is about our original codelab that you can try on…",
-                ),
-                link = "https://medium.com/droidkaigi/droidkaigi-2020%E3%81%A7%E3%81%AE" +
-                    "codelabs%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6-4a677d7456c6",
+                    jaTitle = "DroidKaigiスタッフの@tomoya0x00です。DroidKaigi 2020の開催日が近づいてきましたね。この記事では、DroidKaigi 2020 Day.1に実施する DroidKaigiオリジナルCodelabsについてご紹介します。",
```
Comment (takahirom) [inline model/src/commonMain/kotlin/io/github/droidkaigi/feeder/FeedContents.kt:54]: Can you create FakeFeedContents.kt and exclude it? 🙇 
https://stackoverflow.com/a/46115703/4339442

```
java.lang.AssertionError: Error on line: 54, column: 1
Exceeded max line length (100)
```

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/SpeakersItem.kt [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    Box(modifier = modifier) {
+        speakers.forEachIndexed {i, speaker ->
+            NetworkImage(
+                url = speaker.iconUrl,
+                modifier = Modifier
+                    .offset((14 * i).dp)
+                    .zIndex((-1 * i).toFloat())
+                    .size(18.dp)
+                    .clip(CircleShape)
+                    .border(1.dp, Color.White, CircleShape),
```
Comment (takahirom) [inline uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/SpeakersItem.kt:36]: Awesome technique👍

File: model/src/commonMain/kotlin/io/github/droidkaigi/feeder/FeedContents.kt (hunk 2 above)
Comment (KoutaMatsushita) [reply model/src/commonMain/kotlin/io/github/droidkaigi/feeder/FeedContents.kt]: added rules and create file!

https://github.com/DroidKaigi/conference-app-2021/pull/216/commits/c5094201ee213b4f272dd0751efc838b8a4a2eb3

**Review by takahirom — APPROVED** [review]
Thank you for awesome contribute!

**Pull request merged**